mesonbuild/interpreter/interpreter.py

```diff
@@ -1133,6 +1133,7 @@ def set_backend(self) -> None:
             convertor=lambda x: x[0] if isinstance(x, list) else x,
         ),
         KwargInfo('license', ContainerTypeInfo(list, str), default=['unknown'], listify=True),
+        KwargInfo('license_files', ContainerTypeInfo(list, str), default=[], listify=True),
```

Comment:
I... guess? The thing is that it's:
- always a file, so we don't need to disambiguate like we do for , say, cc.compiles
- never valid as a variable, because you cannot save variables before running `project()`
- longer and wordier, for the same effect

I generally prefer to see `files()` as something explicitly done for cases that might cross a directory context, and not done inline as a kwarg to a function.

Do we really need to support this? I guess if you really want it.